src/index.html

```diff
@@ -1,15 +1,204 @@
+          <label for="formName">
+            Name:
+            <input
+              type="text"
+              name="name"
+              id="formName"
+          >
+          </label>
+        </div>
+        <div class="input">
+          <label for="formAge">
+            How old are You?
+            <input
+              type="number"
+              name="age"
+              value="12"
+              id="formAge"
+              min="1"
+              max="150"
+            >
+          </label>
+        </div>
+        <div class="input">
+          <label for="formData">
+            Full date of birth:
+            <input
+              type="date"
+              name="date"
+              id="formData"
+            >
+          </label>
+        </div>
+        <div class="input">
+          <label for="formTerms">
+            I accept the term of the agreemen
+            <input
+              type="checkbox"
+              name="terms"
+              required
+              id="formTerms"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="section">
+        <legend>
+          Registration
+        </legend>
+        <div class="input">
+          <label for="formEmail">
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+              required
+              id="formEmail"
+            >
+          </label>
+        </div>
+        <div class="input">
+          <label for="formPassword">
+            Password
+            <input
+              type="password"
+              name="password"
+              minlength="4"
+              maxlength="10"
+              required
+              id="formPassword"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="section">
+        <legend>
+          An interesting fact about you!
+        </legend>
+        <div class="input">
+          <label class="input" for="formLove">
+            Do you love cats?
+            <label for="formLove">
+              <input
+                type="radio"
+                name="love"
+                id="formLove"
+              >
+              Yes
+            </label>
+            <label for="formNoLove">
+              <input
+                type="radio"
+                name="love"
+                id="formNoLove"
+              >
+              No
+            </label>
+          </label>
+        </div>
+        <div class="input">
+          <label class="input" for="formColor">
+            What is your favorite color?
+            <input
+              type="color"
+              name="color"
+              id="formColor"
+            >
+          </label>
+        </div>
+        <div class="input">
+          <label class="input" for="formTime">
+            What time do you go to bed?
+            <input
+              type="time"
+              name="time"
```

Comment:
```suggestion
              name="timeToBed"
```
be more specific in naming, as there may be more questions, and some of them may also refer to time or love